everestrs/everestrs_sys/everestrs_sys.cpp

```diff
@@ -80,11 +81,56 @@ JsonBlob Module::call_command(rust::Str implementation_id, rust::Str name, JsonB
     return json2blob(return_value);
 }
 
+void Module::publish_variable(rust::Str implementation_id, rust::Str name, JsonBlob blob) const {
+    handle_->publish_var(std::string(implementation_id), std::string(name),
+                         json::parse(blob.data.begin(), blob.data.end()));
+}
+
 std::unique_ptr<Module> create_module(rust::Str module_id, rust::Str prefix, rust::Str conf) {
     return std::make_unique<Module>(std::string(module_id), std::string(prefix), std::string(conf));
 }
 
-void Module::publish_variable(rust::Str implementation_id, rust::Str name, JsonBlob blob) const {
-    handle_->publish_var(std::string(implementation_id), std::string(name),
-                         json::parse(blob.data.begin(), blob.data.end()));
+namespace {
```

Comment:
there is already an anonymous namespace at the top of the file. Just keep one please.

Reply:
I wanted to keep these functions close the the variant - so they are its immediate apparent why they are here. If you feel strange about 2 namespaces - we can remove it and make the functions static. But I feel the closer these functions are to the variant the easier its to maintain and understand

Reply:
I feel really juck about two anonymous namespaces in one file and static functions are even worse. I appreciate the argument of keeping these close to their use, but I still think it better to move them into the anonymous namespace and only have one area of private stuff in the file. Pretty please with sugar on top? 🐱